Tests/test_image_transform.py

```diff
@@ -173,10 +173,10 @@ def _test_nearest(self, op, mode):
         im = op(im, (40, 10))
 
         colors = im.getcolors()
-        assert colors == [
+        assert set(colors) == {
             (20 * 10, opaque),
             (20 * 10, transparent),
-        ]
+        }
```

Comment:
This is unrelated to the changes made, yes?

Reply:
No, changing the size of the pixels also changed what they get hashed to, so the order of the colors isn't the same. We can ignore the order by comparing them as a set, or sorting both lists before comparing them. All of the colors should be unique so I decided to compare them as a set.

Reply:
Sorting is used in this other test https://github.com/python-pillow/Pillow/blob/a877d7dab2c4c6701d20520187dca94e496b2e7b/Tests/test_image.py#L341-L342 but I don't see that being done anywhere else.